packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2013,7 +2013,10 @@ export default class Meeting extends StatelessWebexPlugin {
     if (meetingInfo && meetingInfo.body && !(meetingInfo.errors && meetingInfo.errors.length > 0)) {
       this.conversationUrl = meetingInfo.body.conversationUrl || this.conversationUrl;
       this.locusUrl = meetingInfo.body.locusUrl || this.locusUrl;
-      this.setSipUri(meetingInfo.body.sipMeetingUri || this.sipUri);
+      this.setSipUri(this.config.experimental.enableUnifiedMeetings ? meetingInfo.body.sipUrl : meetingInfo.body.sipMeetingUri || this.sipUri);
+      if (this.config.experimental.enableUnifiedMeetings) {
+        this.meetingNumber = meetingInfo.body.meetingNumber;
+      }
       this.owner = meetingInfo.body.owner || this.owner;
```

Comment:
according to https://wiki.cisco.com/display/HFWEB/MeetingInfo+API `owner` will be replaced by `hostId`

Reply:
thanks marcin looks like its both , locus sends owner and api send hostID
let me change it

Reply:
added it